cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -153,6 +153,38 @@
         bottom: unit( @grid_gutter-width / 2 / @play-btn-icon-size, em );
     }
 
+    &__center,
+    &__bottom-right {
+        &:focus-visible {
+            outline: none;
+
+            &:before,
+            &::after { 
+                content: '';
+                position: absolute;
+                box-sizing: border-box;
+                display: block;
+            }
+
+            &:before {
+                width: 60px;
+                height: 60px;
+                left: -2px;
+                top: -2px;
+                border: 1px solid @white;
+                outline: 1px solid white;
```

Comment:
So this is my mistake… we should never use color keywords. Either our less variables, hex, or rgba values. So this should be `@white` instead of `white`. I sometimes use `white` in the dev console for testing and this was the result of an erroneous copy/paste.